pyvista/plotting/tools.py

```diff
@@ -1,24 +1,35 @@
 """Module containing useful plotting tools."""
 
 from enum import Enum
+from functools import wraps
 import os
 import platform
 from subprocess import PIPE, Popen, TimeoutExpired
 
 import numpy as np
 
 import pyvista
-
-from . import _vtk
+from pyvista.plotting.axes_actor import AxesActor
+
+from ._vtk import (
+    VTK_ARIAL,
+    VTK_COURIER,
+    VTK_TIMES,
+    vtkAnnotatedCubeActor,
+    vtkMathTextFreeTypeTextRenderer,
+    vtkPolyDataMapper,
+    vtkPropAssembly,
+    vtkRenderWindow,
+)
```

Comment:
nit: I have a significant preference for keeping all VTK imports in the `_vtk` module and accessing them directly from the `_vtk` namespace in your module

Reply:
Thanks for going back to that!